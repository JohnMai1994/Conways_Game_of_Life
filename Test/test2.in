new 10
init 
2 0
2 1
2 2
2 3  
3 0
3 4
4 0
5 1
5 4
-1 -1
print
steps 5
print
steps 2
print
