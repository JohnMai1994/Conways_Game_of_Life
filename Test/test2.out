__XXX_____
__X__X____
__X_______
__X_______
___X_X____
__________
__________
__________
__________
__________
__________
___X_X____
____XX____
____X_____
__________
__________
__________
__________
__________
__________
__________
____X_____
_____XX___
____XX____
__________
__________
__________
__________
__________
__________
